jujugui/static/gui/src/app/user/user.js

```diff
@@ -22,6 +22,17 @@ if (typeof this.jujugui === 'undefined') {
   this.jujugui = {};
 }
 
+// XXX j.c.sackett 2016-05-26 Session expiration should be set via config, but
+// as that also requires an update from blues browser we'll do that in a follow
+// up. For now it's set to the same timeout as in blues browser.
+const EXPIRATION_TIME = 24; // Time for a session, in hours
```

Comment:
As @frankban mentions below I think `EXPIRATION_HOURS` is more descriptive. `EXPIRATION_TIME` sounds like it should be a date.